docs/release/release-2022-08-10.md

```diff
@@ -0,0 +1,308 @@
+# Release Notes August 2022 (2022-08-10)
+
+This release contains **breaking changes**
+
+- Operation parameters without decorators
+- OkResponse is no longer a template
+- Route resolution changes
+- Remove `Map` type
+- `@path` may not decorate optional properties or parameters without a default value
+
+## Operation parameters without decorators
+
+A single undecorated (not marked `@query`, `@header`, `@body` or `@path`) operation parameter will now become a property of the request body rather than have its type define the request body. This allows defining the body with multiple unannotated parameters, which can include unannotated properties that are spread into parameters. (Previously, more than one unannotated parameter was an error.)
+
+For example, the following used to define a request body of type `string`, but now defines a request body that is an object with a property named `body` of type string.
+
+```javascript
```

Comment:
should those not be `cadl`?

Reply:
The formatter was complaining about that, but happy to change

Reply:
We used javascript in the previous version.  At least locally, I get better colorization with this

Reply:
fixed.